packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -714,10 +728,16 @@ class EditorPage {
 		const elementName = isAndroid() ? '//*' : '//XCUIElementTypeOther';
 
 		const locator = `${ elementName }[starts-with(@${ this.accessibilityIdXPathAttrib }, "Use icon button")]`;
-		return await this.driver
-			.elementByXPath( locator )
-			.click()
-			.sleep( isAndroid() ? 200 : 0 );
+		const useIconButton = await waitForVisible( this.driver, locator );
+
+		return await useIconButton.click();
```

Comment:
Similar to my previous comments, I think we could also use here the new helper `clickIfClickable`.